packages/entities/entities-shared/src/components/entity-base-table/EntityBaseTable.vue

```diff
@@ -350,20 +353,29 @@ const handleSortChanged = (sortParams: TableSortParams): void => {
 }
 
 const { setTablePreferences, getTablePreferences } = useTablePreferences()
+
+// Use unique key cacheId (passed down from consuming app and derived from controlPlaneId)
+// for localStorage of user's table preferences across tables, orgs and users
+
+const tablePreferences = ref<UserTablePreferences>(getTablePreferences(cacheId.value))
```

Comment:
suggestion: also use kongponents type here

Reply:
Updated to use kongponents type